pkg/openconnect

```diff
@@ -0,0 +1,37 @@
+[mirrors]
+ftp://ftp.infradead.org/pub/openconnect/openconnect-8.04.tar.gz
```

Comment:
no http link available ?

Reply:
nope, the website only points to that ftp link and when trying to reach it through http you'd get a redirection; also i couldn't find (yet) a mirror for that pkg